Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManager2D.cxx

```diff
@@ -310,6 +310,11 @@ void vtkMRMLMarkupsDisplayableManager2D
 ::ProcessMRMLNodesEvents(vtkObject *caller,unsigned long event,void *callData)
 {
 
+  if (this->GetMRMLScene()->IsBatchProcessing())
+    {
+    return;
+    }
```

Comment:
A greater speedup would be obtained then.

Do you have benchmark: Load time with/without this change ?